_data/site_menu.yml

```diff
@@ -179,65 +190,61 @@ administration:
-
-    - title: ServiceNow
-      path: "/docs/reference/latest/integration_with_servicenow/index.html"
-      desc: A short guide to managing ManageIQ ServiceNow CMDB integration
+    - title: ManageIQ REST API
+      path: "/docs/api"
+      desc: REST API reference
 
 reference:
   title: Reference
   children:
+    - title: Deployment Planning Guide
+      path: "/docs/reference/latest/deployment_planning_guide/index.html"
+      desc: "ManageIQ Deployment Planning Guide"
+    - title: General Configuration
+      path: "/docs/reference/latest/general_configuration/index.html"
+      desc: "ManageIQ General Configuration"
+    - title: High Availability Guide
+      path: "/docs/reference/latest/high_availability_guide/index.html"
+      desc: "ManageIQ High Availability Guide"
+    - title: Managing Authentication for {{ site.data.product.title_short }}
+      children:
+        - title: Active Directory
+          path: "/docs/reference/latest/auth/active_directory.html"
+          desc: "How to manually configure an Appliance’s external authentication to work against Active Directory."
+        - title: 2-Factor-Authentication with IPA
+          path: "/docs/reference/latest/auth/ipa_2fa.html"
+          desc: "How to manually configure an Appliance’s external authentication to work with 2-Factor Authentication with IPA."
+        - title: IPA/AD Trust
+          path: "/docs/reference/latest/auth/ipa_ad_trust.html"
+          desc: "How to manually configure an Appliance’s external authentication to work with AD Trust Authentication using IPA."
+        - title: LDAP
+          path: "/docs/reference/latest/auth/ldap.html"
+          desc: "How to manually configure an Appliance’s external authentication to work against LDAP."
+        - title: SAML
+          path: "/docs/reference/latest/auth/saml.html"
+          desc: "How to manually configure an Appliance’s external authentication to work with SAML."
+        - title: OpenID-Connect
+          path: "/docs/reference/latest/auth/openid_connect.html"
+          desc: "How to manually configure an Appliance’s external authentication to work with OIDC."
+    - title: Appliance Hardening Guide
+      path: "/docs/reference/latest/appliance_hardening_guide/index.html"
+      desc: "ManageIQ Appliance Hardening Guide"
     - title: Capabilities Matrix
       path: "/docs/reference/latest/capabilities_matrix/index.html"
       desc: Capabilities across Providers
-
-    - title: ManageIQ REST API
-      path: "/docs/api"
-      desc: REST API reference
-
     - title: Methods Available for Automation
       path: "/docs/reference/latest/methods_available_for_automation/index.html"
       desc: Advanced automation methods for ManageIQ Management Engine
+    - title: Scripting Actions in ManageIQ
+      path: "/docs/reference/latest/scripting_actions/index.html"
+      desc: "Real-time, bi-directional process integration for ManageIQ Management Engine"
```

Comment:
What's the difference between ManageIQ Management Engine and ManageIQ?  I see both terminology here.  Do we need to normalize on one term?

Reply:
FYI @Fryguy

Reply:
Note, I see this term elsewhere so we can potentially normalize the terms in a followup PR.  It's more obvious here since I see both forms of what I think are the same thing.

Reply:
I bet this used to be CloudForms Management Engine.  Probably best to just use `"Real-time, bi-directional process integration for ManageIQ"`